apps/shared/avifpng.c

```diff
@@ -10,6 +10,13 @@
 #include <stdlib.h>
 #include <string.h>
 
+// See libpng-manual.txt, section XI.
+#if PNG_LIBPNG_VER_MAJOR > 1 || (PNG_LIBPNG_VER_MAJOR == 1 && PNG_LIBPNG_VER_MINOR >= 5)
+#define png_iccp_datap png_bytep
```

Comment:
Nit: Use typedef instead of #define.

Optional: It would be good to avoid the `png_` prefix to make it clear that this is not a libpng type. Since this type is only used within this file, this is not that important.